curator/run_cron.py

```diff
@@ -90,13 +102,30 @@ def run_all_jobs(joblist):
 # run jobs now
 run_all_jobs(my_cron)
 
-thehour = int(os.environ.get('CURATOR_CRON_HOUR', 0))
-theminute = int(os.environ.get('CURATOR_CRON_MINUTE', 0))
+thehour = decoded.get('.defaults', {}).get('runhour', None)
+if not thehour:
+    thehour = os.getenv('CURATOR_RUN_HOUR', None)
+if not thehour:
```

Comment:
I'm not sure if this is possible in python, but couldn't you collapse this into a single statement and have something like `thehour = os.getenv('CURATOR_RUN_HOUR', defaults.get('runhour', 0))` ?